R/gumbel_mean.R

```diff
@@ -0,0 +1,171 @@
+# Euler-Mascheroni constant used in a few equations in Gumbel
+euler_mascheroni <- 0.57721566490153
```

Comment:
use `-digamma(1)` instead of this global variable thing.

Reply:
Substituted global with -digamma. Added comment at first appearence of the constant `-digamma(1) == Euler-Mascheroni constant`.